.github/workflows/gradle-smoke-test.yml

```diff
@@ -3,34 +3,25 @@ name: Gradle smoke test
 on: [push]
 
 jobs:
-    build-ccud:
-        runs-on: ubuntu-latest
-
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v2
-            - name: Set up JDK 11
-              uses: actions/setup-java@v2
-              with:
-                  java-version: '11'
-                  distribution: 'adopt'
-            - name: Build CCUD Gradle plugin
-              uses: gradle/gradle-build-action@v2
-              with:
-                  arguments: build -x signArchives
-                  build-root-directory: common-custom-user-data-gradle-plugin
 
-    verify-samples:
+    verify-gradle-samples:
         runs-on: ubuntu-latest
+
         strategy:
             matrix:
                 include:
-                    - path-to-sample-build-file: "build-data-capturing-gradle-samples/capture-dependency-resolution/gradle-dependency-resolution.gradle"
-                    - path-to-sample-build-file: "build-data-capturing-gradle-samples/capture-git-diffs/gradle-git-diffs.gradle"
-                    - path-to-sample-build-file: "build-data-capturing-gradle-samples/capture-os-processes/gradle-os-processes.gradle"
-                    - path-to-sample-build-file: "build-data-capturing-gradle-samples/capture-quality-check-issues/gradle-quality-check-issues.gradle"
-                    - path-to-sample-build-file: "build-data-capturing-gradle-samples/capture-slow-workunit-executions/gradle-slow-task-executions.gradle"
-                    - path-to-sample-build-file: "build-data-capturing-gradle-samples/capture-test-execution-system-properties/gradle-test-execution-system-properties.gradle"
+                    - build-dir: "common-gradle-enterprise-gradle-configuration"
```

Comment:
Where do we test the `common-custom-user-data-gradle-plugin`?

Reply:
We are building it in the "Publish current snapshot of the CCUD Gradle plugin to local Maven repository" step and then using it on the samples

Reply:
Ah that makes sense since there aren't any tests in there.